descriptors/ModuleDescriptor-template.json

```diff
@@ -51,6 +51,11 @@
           "pathPattern": "/search/authorities",
           "permissionsRequired": [ "search.authorities.collection.get" ]
         },
+        {
+          "methods": [ "GET" ],
+          "pathPattern": "/search/authorities/facets",
+          "permissionsRequired": [ "search.authorities.facets.collection.get" ]
+        },
         {
           "methods": [ "GET" ],
           "pathPattern": "/search/instances/facets",
```

Comment:
Should we generalize this endpoint and passing parameter "recodType". 
Instead of creating two absolutely identical endpoints